### DISCUSSION TITLE
errCode = 7, detailMessage = table type is not OLAP, tableName=out_eqp_deviceinfo, type=ODBC

### DISCUSSION THREAD

**Discussion by wuchencm** [apache/doris#11003] [RFC]
页面报错，但是select 能查询出数据，使用的是postgresql的外表，doris 1.1.0
errCode = 7, detailMessage = table type is not OLAP, tableName=out_eqp_deviceinfo, type=ODBC
![image](https://user-images.githubusercontent.com/14210543/179704982-f2682857-dbe1-4147-bacf-d536b721edfa.png)

**Comment by JNSimba** [comment]
Execute it under the MySQL command line and try, the web page should not support querying ODBC tables

**Reply by wuchencm** [reply]
get it

**Reply by JNSimba** [reply]
I verified that the web page can query the odbc table. The above error is because doris cannot query the schema information of the odbc table, and the error is reported when calling the schema api
